### DISCUSSION TITLE
Style sheet in child theme does not work

### DISCUSSION THREAD

**Discussion by ArdH** [liedekef/events-made-easy#106] [General]
Hi Franky,
I have the impression (for quit a long time/about a couple of monthes) that an extra style sheet eme.css that I put in the child theme directory does not work.
I took some hours to solve this myself.
I found that when I make two changes in the file eme-actions.php the extra style sheet is working again as intended by me.
The changes are removing 
`, EME_VERSION`
on line 503 and on line 632 (plugin version 2.3.58)
Is this correct, or is there something else to be changed?
Thanks for this nice plugin and your answer on this question.
Ard Hartsuijker www.camras.nl

**Comment by liedekef** [comment]
Indeed, it seems some arguments were not correct. Can you check if the just-updated version of eme_actions.php is better?
https://github.com/liedekef/events-made-easy/blob/main/eme-actions.php

**Reply by ArdH** [reply]
The eme.css file in the theme child directory is now working and all minor changes in style (I made) are visible again.

Thank you for your quick repair, Ard Hartsuijker
